src/pages/CreateDesign/components/StepProgressBar/index.tsx

```diff
@@ -0,0 +1,99 @@
+import Step from '@material-ui/core/Step';
+import StepConnector from '@material-ui/core/StepConnector';
+import StepLabel from '@material-ui/core/StepLabel';
+import Stepper from '@material-ui/core/Stepper';
+import { makeStyles, withStyles } from '@material-ui/core/styles';
+import clsx from 'clsx';
+import { currentStepAtom } from 'pages/CreateDesign/recoils';
+import React from 'react';
+import { useRecoilValue } from 'recoil';
+
+const Connector = withStyles({
+  alternativeLabel: {
+    top: 10,
+    left: 'calc(-50%)',
+    right: 'calc(50%)',
+  },
+  active: {
+    '& $line': {
+      borderColor: '#e0562e',
+    },
+  },
+  completed: {
+    '& $line': {
+      borderColor: '#e0562e',
+    },
+  },
+  line: {
+    borderColor: '#dcdcdc',
+    borderTopWidth: 5,
+  },
+})(StepConnector);
+
+const StepIconStyles = makeStyles({
+  root: {
+    color: '#dcdcdc',
+    display: 'flex',
+    height: 22,
+    zIndex: 1,
+    alignItems: 'center',
+  },
+  active: {
+    color: '#e0562e',
+  },
+  circle: {
+    width: 15,
+    height: 15,
+    borderRadius: '50%',
+    backgroundColor: 'currentColor',
+  },
+  completed: {
+    color: '#e0562e',
+    fontSize: 18,
+  },
+});
+
+interface StepIconProps {
+  active: boolean;
+  completed: boolean;
+}
+
+function StepIcon(props: StepIconProps) {
+  const classes = StepIconStyles();
+  const { active, completed } = props;
+
+  return (
+    <div
+      className={clsx(classes.root, {
+        [classes.active]: active,
+      })}
+    >
+      {completed ? (
+        <div className={clsx(classes.circle, classes.completed)} />
+      ) : (
+        <div className={classes.circle} />
+      )}
```

Comment:
```suggestion
       <div className={clsx(classes.circle, {[classes.completed]: completed})} />
```
이런식으로 할 수 있는걸로 알고있습니다!
그나저나 이런식으로 스타일을 주면 scss 쓰는 느낌인 것 같은데... 이런식으로 구현하는게 아니라 styledcomponent 형식처럼 구현해야하지 않을까요?

Reply:
오버라이딩 해버리면 안되나욤?

Reply:
한유리님이 제안해주신 방법으로 일단 반영했어요! https://github.com/k-roffle/knitting-frontend/pull/11/commits/b5dff6b5c16bcd341fa450bd8b0832050e62a618

스타일 주입 방식 변경하는 건 #13 에서 같이 진행하겠습니다!